### PR TITLE
Store list (LIN-24, LIN-25)

### PR DESCRIPTION
**Pull request by FilPag**
- **CardTile** - Clickable card with, banner, title and description which redirects to page
- **SearchBox** - Stylized input box
- **StoreListPage** - Page with listing an overview of all stores. can currently be tested under ``/store_list_test``

Resolves LIN-24, LIN-25

### CODE REVIEW

File: frontend/web/src/components/CardTile/CardTile.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  to?: string,
+}
+
+
+const CardTile = ({className, description, title, image, to = ''}: CardTileProps) => {
+  return (
+    <Link to={to}
+    className={styles.container + ' ' + className}
+    >
+      <img src={image} className={styles.banner}/>
```
Comment (gali8033) [inline frontend/web/src/components/CardTile/CardTile.tsx:19]: kanske lägga till alt-attribut för bättre UX 😃

**Review by gali8033 — APPROVED** [review]
Såg jättebra ut!

File: frontend/web/src/components/CardTile/CardTile.tsx (hunk 1 above)
Comment (gali8033) [inline frontend/web/src/components/CardTile/CardTile.tsx:19]:
```suggestion
      <img src={image} className={styles.banner} alt='Banner' />
```

File: frontend/web/src/components/Docs.md [hunk 2]
```diff
@@ -7,4 +7,25 @@
 ### Props
 
 ## AuthenticationForm
-### Props
\ No newline at end of file
+### Props
+
+## CardTile
+Clickable card which redirects the user to the specified link
+### Porps
```
Comment (sebastianfallman) [inline frontend/web/src/components/Docs.md:14]: GIEF PORPS
```suggestion
### Props
```

**Pull request merged**